### PR TITLE
BasP-MR-16: GetAll - Post - Put - Validation 

### PR DESCRIPTION
**Pull request by leo-balonchard**
Work on the following member methods:

- Get All members 
- Create member
- Update member

Also perform the following validation:

- Validation for new member creation

### CODE REVIEW

File: src/routes/members.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const membersController = require('../controllers/members');
+const validations = require('../validations/members');
+
+const router = express.Router();
+
+router
+  .get('/', membersController.getAllMembers)
+  .post('/', validations.validateMemberCreation, membersController.createMember)
+  .put('/:id', validations.validateMemberCreation, membersController.updateMember);
```
Comment (leo-balonchard) [inline src/routes/members.js:10]: I'm going to switch to update validation in the merge (this is because my teammate has that validation).

**Review by fernandofernandezuy — APPROVED** [review]
Seems good

**Review by matiasluispeter — APPROVED** [review]
Seems good to me!

**Review by NicolasKler — APPROVED** [review]
Looks great to me

**Review by Juanoauil — APPROVED** [review]

**Pull request merged**